vizro-core/src/vizro/static/css/vizro-bootstrap.min.css

```diff
@@ -1,5 +1,5 @@
verted: #b05000;--sequentialOrange-700: #973a00;--sequentialOrange-700-inverted: #973a00;--sequentialOrange-800: #7e2400;--sequentialOrange-800-inverted: #7e2400;--sequentialOrange-900: #640d00;--sequentialOrange-900-inverted: #640d00;--sequentialIndigo-100: #dfd8fa;--sequentialIndigo-100-inverted: #dfd8fa;--sequentialIndigo-200: #c3c1ed;--sequentialIndigo-200-inverted: #c3c1ed;--sequentialIndigo-300: #aba8e0;--sequentialIndigo-300-inverted: #aba8e0;--sequentialIndigo-400: #9390d2;--sequentialIndigo-400-inverted: #9390d2;--sequentialIndigo-500: #7a79c4;--sequentialIndigo-500-inverted: #7a79c4;--sequentialIndigo-600: #6163b5;--sequentialIndigo-600-inverted: #6163b5;--sequentialIndigo-700: #474ea6;--sequentialIndigo-700-inverted: #474ea6;--sequentialIndigo-800: #2a3994;--sequentialIndigo-800-inverted: #2a3994;--sequentialIndigo-900: #002680;--sequentialIndigo-900-inverted: #002680;--sequentialYellow-100: #fff7cd;--sequentialYellow-100-inverted: #fff7cd;--sequentialYellow-200: #ffed9b;--sequentialYellow-200-inverted: #ffed9b;--sequentialYellow-300: #ffe16a;--sequentialYellow-300-inverted: #ffe16a;--sequentialYellow-400: #ffd545;--sequentialYellow-400-inverted: #ffd545;--sequentialYellow-500: #ffc107;--sequentialYellow-500-inverted: #ffc107;--sequentialYellow-600: #dba005;--sequentialYellow-600-inverted: #dba005;--sequentialYellow-700: #b78103;--sequentialYellow-700-inverted: #b78103;--sequentialYellow-800: #936402;--sequentialYellow-800-inverted: #936402;--sequentialYellow-900: #7a4f01;--sequentialYellow-900-inverted: #7a4f01;--sequentialTeal-100: #a5eae8;--sequentialTeal-100-inverted: #a5eae8;--sequentialTeal-200: #7dd5d3;--sequentialTeal-200-inverted: #7dd5d3;--sequentialTeal-300: #5ebfbc;--sequentialTeal-300-inverted: #5ebfbc;--sequentialTeal-400: #44a8a6;--sequentialTeal-400-inverted: #44a8a6;--sequentialTeal-500: #2e9190;--sequentialTeal-500-inverted: #2e9190;--sequentialTeal-600: #1b7b7a;--sequentialTeal-600-inverted: #1b7b7a;--sequentialTeal-700: #0c6565;--sequentialTeal-700-inverted: #0c6565;--sequentialTeal-800: #025050;--sequentialTeal-800-inverted: #025050;--sequentialTeal-900: #003b3c;--sequentialTeal-900-inverted: #003b3c;--sequentialRed-100: #f8d6da;--sequentialRed-100-inverted: #f8d6da;--sequentialRed-200: #fcb6ba;--sequentialRed-200-inverted: #fcb6ba;--sequentialRed-300: #f8989b;--sequentialRed-300-inverted: #f8989b;--sequentialRed-400: #ed7b7f;--sequentialRed-400-inverted: #ed7b7f;--sequentialRed-500: #dd6065;--sequentialRed-500-inverted: #dd6065;--sequentialRed-600: #c9474c;--sequentialRed-600-inverted: #c9474c;--sequentialRed-700: #b22f36;--sequentialRed-700-inverted: #b22f36;--sequentialRed-800: #981822;--sequentialRed-800-inverted: #981822;--sequentialRed-900: #7d000f;--sequentialRed-900-inverted: #7d000f;--surfaces-bg-card: #F5F6F6;--surfaces-bg-card-inverted: #232632;--bs-primary-rgb: #141721e0;--bs-secondary-color: #14172199;--bs-nav-link-color: #14172199;--bs-form-valid-color: #26bf56ff;--bs-form-invalid-color: #f03b3aff}/*!
```

Comment:
In theory, the original issues are fixed by `self.dash.index_string = self.dash.index_string.replace("<html>", "<html data-bs-theme='dark'")` as @antonymilne  suggested. However, I would still add the current bootstrap changes in, as the default bg colour does not match exactly. The bootstrap theme changes ensure the right bg coloring and fixes some specificity issues as well.